byron/ledger/formal-spec/properties.tex

```diff
@@ -156,3 +156,76 @@ \subsection{UTxO Properties}
   $$
 
 \end{property}
+
+\subsection{PVBUMP STS Properties}
+\label{sec:pvbump-properties}
+
+Property~\ref{prop:pvbump-no-proposals} states that when no proposals are
+available, the system remains in the same state it started with.
+
+\begin{property}[PVBUMP on no proposals]\label{prop:pvbump-no-proposals}
+  \textrm{For all environments that have an empty proposal list, the PVBUMP
+    STS stays in the same state:}
+
+  \inference
+  {
+    \var{fads} = \epsilon
+  }
+  {
+    {\left(\begin{array}{l}
+             s_n\\
+             \var{fads}
+           \end{array}\right)}
+       \vdash
+       {
+         \left(
+           \begin{array}{l}
+             \var{pv}, \var{pps}\\
+           \end{array}
+         \right)
+       }
+       \trans{pvbump}{}
+       {
+         \left(
+           \begin{array}{l}
+             \var{pv}, \var{pps}\\
+           \end{array}
+         \right)
+       }
+     }
+\end{property}
+
+Property~\ref{prop:pvbump-early-on} states that early on in the lifetime
+of the blockchain the PVBUMP system remains in the same state it started with.
+
+\begin{property}[PVBUMP early on]\label{prop:pvbump-early-on}
+  \textrm{For all current slots that have an index less than or equal to
```

Comment:
Maybe this prose can be outside of the property.

Reply:
Fixed!